Engine/csound_orc.lex

```diff
@@ -77,7 +77,7 @@ int get_next_char(char *, int32_t, struct yyguts_t*);
 %option nounput
 
 IDENT           [a-zA-Z_][a-zA-Z0-9_]*
-TYPED_IDENTIFIER  [a-zA-Z_][a-zA-Z0-9_]*:[a-zA-Z_][a-zA-Z0-9_]*
+TYPED_IDENTIFIER  [a-zA-Z_][a-zA-Z0-9_@]*:[a-zA-Z_][a-zA-Z0-9_]*
```

Comment:
I think this rule is off as it permits multiple @ symbols. I need to check but it should probably use a ? for optional @global, something like :

[a-zA-Z_][a-zA-Z0-9_]*(@global)?:[a-zA-Z_][a-zA-Z0-9_]*

Reply:
you are right, improved.